micromamba/tests/test_info.py

```diff
@@ -80,19 +80,20 @@ def test_not_env(tmp_home, tmp_root_prefix, prefix_selection, existing_prefix):
 
 
 @pytest.mark.parametrize("base_flag", ["", "--base"])
+@pytest.mark.parametrize("envs_flag", ["", "-e", "--envs"])
 @pytest.mark.parametrize("json_flag", ["", "--json"])
 @pytest.mark.parametrize("prefix_selection", [None, "prefix", "name"])
-def test_base_subcommand(
-    tmp_home, tmp_root_prefix, prefix_selection, base_flag, json_flag, monkeypatch
+def test_subcommand(
```

Comment:
The name of the test should stay as it is (i.e `test_base_subcommand`), because we are testing the base environment (`-n base` equivalent to `-p root_prefix`).
We should add `envs_flag` to other tests as well when relevant (`test_env` for example, maybe even all of them in the test file, I didn't check everything).